Arkadzi_Tarkan/15.Jenkins.Start/node/authorized_keys

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCnvIDelQ/waq8EzayNuDav+kNIyNvTQPEWctbN3Ha5/EPGSJoDjFnwId7h94xRrYwLqTqmLDE38uLk63nnjiFnC3YETX7LcJf0t5jJGc1Fh77cAWn1zyz65FLK9IeuPrXSeLqNOQYIVAzang3DaxdqY8AFRoE31T51A2+s4EyFqODWQVa6+5s8RAFk3FIR4m4cQQzt2VJEedo5v2Bkvy0WyF+hg33MsBkSqxH8z3eLVVfoS86zN/a+Q7bYL9Phnk0BpigBIThSUJ1a+2UmUSlBT9cQUccbQk8lXaS7phwkFNQWYA5IbZxGJjbkwt8PCPSAX/I1lpmNoEklM1eyBCCP jenkins@localhost.localdomain
```

Comment:
please remove it, then I can merge

Reply:
del authorized_keys_file and 11.Ansible.start